pcl_conversions/package.xml

```diff
@@ -5,6 +5,9 @@
   <description>Provides conversions from PCL data types and ROS message types</description>
 
   <author email="william@osrfoundation.org">William Woodall</author>
+  <author email="paul@bovbel.com">Paul Bovbel</author>
```

Comment:
Adding maintainers?

Reply:
Author != maintainer .  Adding authors is generally fine, adding maintainers is generally a no-no :).

Reply:
Got it, I just looked at this and said "huh, why are you adding _other_ people unrelated?"